id/news/_posts/2018-03-28-large-request-dos-in-webrick-cve-2018-8777.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: news_post
+title: "CVE-2018-8777: DoS karena permintaan yang besar pada WEBrick"
+author: "usa"
+translator: "meisyal"
+date: 2018-03-28 14:00:00 +0000
+tags: security
+lang: id
+---
+
+Ada sebuah kerentanan *out-of-memory* DoS dengan sebuah permintaan yang besar
+pada WEBrick yang di-*bundle* bersama dengan Ruby. Kerentanan ini telah
```

Comment:
“bundled with Ruby” should be dikemas dalam Ruby IMO

Reply:
or dikemas dengan Ruby

Reply:
I prefer "dikemas dengan Ruby". I just changed to that words.

Reply:
I prefer "dikemas dengan Ruby". I just changed it to that words.

Reply:
thanks